openml/tasks/task.py

```diff
@@ -200,6 +208,14 @@ def publish(self) -> int:
 
 
 class OpenMLSupervisedTask(OpenMLTask, ABC):
+    """OpenML Supervised Classification object.
+
+       Inherited from :class:`openml.OpenMLTask`
+
+       Parameters
+       ----------
+       target_name : str
```

Comment:
Could you please add some further explanation, like `Name of the target feature used for supervised learning`?

Reply:
Added.